disnake/ext/commands/bot.py

```diff
@@ -23,9 +23,9 @@
 DEALINGS IN THE SOFTWARE.
 """
 
-from __future__ import annotations
+# from __future__ import annotations
```

Comment:
this might've been an accidental change?

Reply:
Huh, somehow happened during the master merge, yeah. Will fix now.